engine/polyglot-api/src/main/scala/org/enso/polyglot/Suggestion.scala

```diff
@@ -332,6 +377,26 @@ object Suggestion {
       s",documentation=" + (if (shouldMask) documentation.map(_ => STUB)
                             else documentation) +
       s",reexports=$reexports)"
+
+    override def withReturnType(returnType: String): Suggestion =
+      copy(returnType = returnType)
+
+    override def update(
```

Comment:
Why is this method completely the same as the same method on line 476?

Reply:
There is a small amount of DRY in this case but I would prefer to keep it that way rather than overriding a copy method generated for case-classes.